docs/content/guides/migrations/from-step-launchers-to-pipes.mdx

```diff
@@ -0,0 +1,333 @@
+---
+title: "Migrating From Step Launchers to Dagster Pipes | Dagster Docs"
+description: "Learn how to migrate from step launchers to Dagster Pipes in Dagster."
+---
+
+# Migrating From Step Launchers to Dagster Pipes
+
+In this guide, we’ll show you how to migrate from using step launchers to using [Dagster Pipes](/concepts/dagster-pipes) in Dagster.
+
+The reason for this migration and a detailed comparison between the two alternatives can be found in [this GitHub Discussion](https://github.com/dagster-io/dagster/...).
```

Comment:
Is this available?

Reply:
Not yet. I guess I'll add the exact URL once it's posted. 

Or should we create the GitHub Discussion before merging this guide? @yuhan

Reply:
Decided to remove this for now